modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/search.query/20_ngram_search_field_match_only.yml

```diff
@@ -0,0 +1,144 @@
+"ngram search":
```

Comment:
Super nitpick...it seems like most of these match_only_text-related YAML test files end in "match_only_text.yml", but some end in "match_only.yml". Probably good to be consistent.